lib/repl.js

```diff
@@ -519,16 +519,20 @@ function REPLServer(prompt,
 
     // Check to see if a REPL keyword was used. If it returns true,
     // display next prompt and return.
-    if (cmd && cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
-      var matches = cmd.match(/^\.([^\s]+)\s*(.*)$/);
-      var keyword = matches && matches[1];
-      var rest = matches && matches[2];
-      if (self.parseREPLKeyword(keyword, rest) === true) {
-        return;
-      } else if (!self.bufferedCommand) {
-        self.outputStream.write('Invalid REPL keyword\n');
-        finish(null);
-        return;
+    if (cmd) {
+      if (cmd.charAt(0) === '.' && cmd.charAt(1) !== '.' &&
+          isNaN(parseFloat(cmd))) {
+        const matches = cmd.match(/^\.([^\s]+)\s*(.*)$/);
+        const keyword = matches && matches[1];
+        const rest = matches && matches[2];
+        if (self.parseREPLKeyword(keyword, rest) === true) {
+          return;
+        }
+        if (!self.bufferedCommand) {
```

Comment:
AFAICT for `(cmd == false & self.bufferedCommand == true)` it will act differently.
I think since for `v6.x` stability if very important, we should keep the original `if` (as long as it still passes the test.)
I started a CI job to see if any test will fail. If you can think if a test case that creates `(cmd == false & self.bufferedCommand == true)` maybe we should add it.